Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoISE.yaml

```diff
@@ -0,0 +1,70 @@
+Parser:
```

Comment:
@spsocprime Please update union ASIM and VIM parser files in this PR

Reply:
Added records to union parsers